tests/framework/extensions/rke1/nodetemplates/azure/create.go

```diff
@@ -0,0 +1,30 @@
+package nodetemplates
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/rke1/nodetemplates"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+)
+
+const azureNodeTemplateNameBase = "azureNodeConfig"
+
+// CreateAzureNodeTemplate is a helper function that takes the rancher Client as a parameter and creates
+// a Azure node template credential, and returns the NodeTemplate response
```

Comment:
Just a nitpick, for our documentation it should read `an Azure node template` remove the credential